.tools/readmes/service_readme.jinja2

```diff
@@ -1,6 +1,11 @@
-<!--Generated by WRITEME on {{ now() }} (UTC)-->
-{% import 'includes/macros.jinja2' as macs %}
 # {{ service['short'] }} code examples for the {{ sdk['short'] }}
+{% import 'includes/macros.jinja2' as macs %}
+{% if deprecated %}
+
+> NOTE: This API has been deprecated.
```

Comment:
Same thing about "deprecated" here. Otherwise, the sentiment of no longer maintaining these examples is good.